SPECS/rsyslog/rsyslog.spec

```
@@ -11,7 +13,8 @@ Source0:        http://www.rsyslog.com/files/download/rsyslog/%{name}-%{version}
 Source1:        rsyslog.service
 Source2:        50-rsyslog-journald.conf
 Source3:        rsyslog.conf
-Source4:        http://www.rsyslog.com/files/download/rsyslog/%{name}-doc-%{version}.tar.gz
+# Upstream only publishes built docs for base_version.0
+Source4:        http://www.rsyslog.com/files/download/rsyslog/%{name}-doc-%{base_version}.0.tar.gz
```

Let's prefer `https` wherever we can :)
```suggestion
Source4:        https://www.rsyslog.com/files/download/rsyslog/%{name}-doc-%{base_version}.0.tar.gz
```